ocaml/idl/datamodel.ml

```diff
@@ -857,6 +859,8 @@ let _ =
     ~doc:"The hosts in this pool are not compatible." ();
   error Api_errors.pool_hosts_not_homogeneous [ "reason" ]
     ~doc:"The hosts in this pool are not homogeneous." ();
+  error Api_errors.pool_management_cannot_be_set_on_system_network [ "network" ]
+    ~doc:"You tried to configure pool management on system network: It is not allowed." ();
```

Comment:
I'm not a native speaker but I think this should read like one of these - depending on what is intended:

* "You tried to configure pool management on the system network: It is not allowed."
* "You tried to configure pool management on the system's network: It is not allowed."